src/Features/Core/Portable/InitializeParameter/AbstractInitializeParameterCodeRefactoringProvider.cs

```diff
@@ -93,6 +94,7 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
                 return;
             }
 
+            RoslynDebug.Assert(parameter.ContainingSymbol is not null);
```

Comment:
Consider using regular `Debug.Assert`.

Reply:
There's a few instances of this in the PR, I'm not going to call out every one.

Reply:
The places I used `RoslynDebug.Assert` vs. `Debug.Assert` are where the project does not have a .NET Core target framework. In that setup `Debug.Assert` offers no nullable value and hence I need to use `RoslynDebug.Assert`. If you see any places I used `RoslynDebug` in a .NET core project happy to fix those but for the others I'm pretty much forced to use `RoslynDebug.Assert`

Reply:
Ah, I didn't realize that was still a thing in parts of our codebase.